test/integration/route-indexes/test/index.test.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-env jest */
+/* global jasmine */
+import { join } from 'path'
+import {
+  findPort,
+  launchApp,
+  killApp,
+  nextBuild,
+  nextStart,
+  fetchViaHTTP,
+} from 'next-test-utils'
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000 * 60 * 2
+
+let app
+let appPort
+const appDir = join(__dirname, '../')
+
+const runTests = () => {
+  it('should handle / correctly', async () => {
+    const res = await fetchViaHTTP(appPort, '/')
+    expect(res.status).toBe(200)
+    expect(await res.text()).toContain('hello from index')
+  })
+
+  it('should handle /index correctly', async () => {
+    const res = await fetchViaHTTP(appPort, '/index')
+    expect(res.status).toBe(200)
+    expect(await res.text()).toContain('hello from index')
+  })
+
+  it('should handle /nested-index/index correctly', async () => {
```

Comment:
Can we add tests for what we expect for `/nested-index` and `/nested-index/index/index` (ditto for other routes)?

Reply:
Added